core/src/main/scala/org/http4s/otel4s/middleware/ClientMiddleware.scala

```diff
@@ -1,4 +1,4 @@
-package io.chrisdavenport.http4sotel4s
+package org.http4s.otel4s.middleware
```

Comment:
Design pondering: does this need to be in a `middlware` subpackage.

Yes, if we're going to put other things that aren't middleware in `org.http4s.otel4s`.  Maybe some common helpers for instrumenting http4s backends?  Hard to say before we begin the Ember effort _and_ have a second one.

No, if this is it, we can flatten things out.

If nobody is sure, the way it is is probably better, because it leaves us room in the namespace if we need it later.

Reply:
> If nobody is sure, the way it is is probably better, because it leaves us room in the namespace if we need it later.

this was my reasoning. and the cost of having it in a slightly deeper package is quite low